crates/bevy_ecs/src/identifier/mod.rs

```diff
@@ -6,13 +6,16 @@
 #[cfg(feature = "bevy_reflect")]
 use bevy_reflect::Reflect;
 
-use self::{error::IdentifierError, kinds::IdKind, masks::IdentifierMask};
+use self::masks::IdentifierMask;
 use std::{hash::Hash, num::NonZeroU32};
 
 pub mod error;
 pub(crate) mod kinds;
 pub(crate) mod masks;
 
+pub use self::error::IdentifierError;
+pub use self::kinds::IdKind;
```

Comment:
Sneaking this in, in part for testing. It didn't make much sense for `Identifier` and `Identifier::new` to both be `pub`, but its argument/return types not.

Reply:
Don't actually need this for the tests anymore, so I'll remove.